android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java

```diff
@@ -93,6 +93,14 @@ protected void onCreate(Bundle savedInstanceState) {
         sessionsRecyclerView.setHasFixedSize(true);
         gridLayoutManager = new GridLayoutManager(this, spanCount);
         sessionsRecyclerView.setLayoutManager(gridLayoutManager);
+
+        //removing sessions with draft state
+        for(int i=0; i<mSessions.size(); i++) {
+            if(mSessions.get(i).getState().equals("draft")) {
+                mSessions.remove(i);
+            }
+        }
```

Comment:
This won't handle when the sessions are updated through Realm, and you are removing when the list is empty since the data is not necessarily loaded